forc/src/cli/commands/new.rs

```diff
@@ -23,9 +23,15 @@ pub struct Command {
     /// Adding this flag creates an empty workspace.
     #[clap(long)]
     pub workspace: bool,
-    /// Set the package name. Defaults to the directory name
+    /// Set the package name. Defaults to the directory name.
     #[clap(long)]
     pub name: Option<String>,
+    /// The path at which the standard library is located.
+    /// Does not override the std dependency if missing.
```

Comment:
```suggestion
    /// If unspecified, the std will be implicitly included as a git dependency
    /// via the git tag that matches `forc`'s version.
```

Perhaps it's worth clarifying that we're talking about the case where `stdlib` is *unspecified* like this? Otherwise, "missing" might be misinterpreted as though we fall back if the given std path doesn't exist?